lib/CPathfinder.cpp

```diff
@@ -19,6 +19,86 @@
 #include "CConfigHandler.h"
 #include "../lib/CPlayerState.h"
 
+bool canSeeObj(const CGObjectInstance * obj)
```

Comment:
This method is a bit awkward. Is not it a responsibility of CGObjectInstance or something around it?